content/plugins/aggressive-splitting-plugin.md

```diff
@@ -0,0 +1,23 @@
+---
+title: AggressiveSplittingPlugin
+contributors:
+  - 
+---
+
+
+```
+new webpack.optimize.AggressiveSplittingPlugin(options),
+```
+
+## Options
+```
+{
+  minSize: 30000, //Byte, split point. Default: 30720
+  maxSize: 50000, //Byte, maxsize of per file. Default: 51200
+  chunkOverhead: 0, //Default: 0
+  entryChunkMultiplicator: 1, //Default: 1
+}
+```
+
+## Examples

```

Comment:
You'll need to add line breaks between the header and code block here and above.